test/libdrampowertest/commands.trace

```diff
@@ -16,4 +16,4 @@
 210,RDA,0
 232,ACT,4
 247,WRA,4
-248,PDN_F_ACT,3
+265,PDN_F_PRE,3
```

Comment:
The last command in the file commands.trace was different from the last command that is done in the file lib_test.cc.